docs/sources/reference/commandline/cli.md

```diff
@@ -17,6 +17,31 @@ or execute `docker help`:
 
       ...
 
+## Environment Variables
+
+For easy reference, the follow list of environment variables will be used by
+the `docker` command line:
+
+* `DOCKER_CERT_PATH` The location of your authentication keys.
+* `DOCKER_DRIVER` The graph driver to use.
+* `DOCKER_HOST` Daemon socket to connect to.
+* `DOCKER_NOWARN_KERNEL_VERSION` If set docker will not print a warning about your Linux kernel being unsuitable for Docker.
+* `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
+* `DOCKER_TLS_VERIFY` If set this will use TLS and verify the remote.
```

Comment:
When set Docker uses TLS and verifies the remote.

or 

When set Docker uses TLS to verify the remote.